src/__tests__/lib/exampleData.js

```diff
@@ -453,15 +453,75 @@ export const makeOutboxMessage = (data: $Shape<$Diff<Outbox, {| id: mixed |}>>):
 
 const privateReduxStore = createStore(rootReducer);
 
-/** The global Redux state, at its initial value. */
+/**
```

Comment:
> example data: Add a batteries-included version of the Redux state.

Cool, sounds good! Linking some prior brainstorming about this: https://github.com/zulip/zulip-mobile/pull/4299#discussion_r518437296.

Reply:
Ah, thanks for locating that :slightly_smiling_face: Rereading, it looks like this covers all the thoughts there, either directly or as TODO comments; so that's reassuring.